src/RNTupleWriter.cc

```diff
@@ -1,20 +1,24 @@
 #include "podio/RNTupleWriter.h"
+
+#include "TFile.h"
 #include "podio/CollectionBase.h"
+#include "podio/CollectionBuffers.h"
 #include "podio/DatamodelRegistry.h"
-#include "podio/GenericParameters.h"
+#include "podio/Frame.h"
 #include "podio/SchemaEvolution.h"
 #include "podio/podioVersion.h"
 #include "rootUtils.h"
 
-#include "TFile.h"
-
+#include <ROOT/REntry.hxx>
+#include <ROOT/RError.hxx>
 #include <ROOT/RField.hxx>
-#include <ROOT/RNTuple.hxx>
 #include <ROOT/RNTupleModel.hxx>
 
-#include <algorithm>
+#include <Compression.h>
```

Comment:
Where does `Compression.h` come from?

Reply:
It's a file that ends up in the `include/` of ROOT, no idea why it gets pulled. Removed.